config/routes.rb

```diff
@@ -13,6 +13,7 @@
   match 'settings' => 'settings#index'
   match 'settings/save' => 'settings#save'
   match 'settings/lastfm_callback' => 'settings#lastfm_callback'
+  match 'settings/lastfm_connect' => 'settings#lastfm_connect'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.